diff --git a/dist/keystar-ui-typography.js b/dist/keystar-ui-typography.js
index 9e97e5ae5a2197e5ece9b73aa75ea518cfd58dd6..82c6224fa0f6cb314dfc527d0282b5790eec682c 100644
--- a/dist/keystar-ui-typography.js
+++ b/dist/keystar-ui-typography.js
@@ -100,7 +100,7 @@ const overflowMap = {
   },
   breakword: {
     // hyphens: 'auto', // too eager
-    overflowWrap: 'break-word'
+    overflowWrap: 'normal'
   }
 };
 
@@ -522,8 +522,8 @@ function useProseStyleProps(props) {
       minHeight: 0,
       minWidth: 0,
       position: 'relative',
-      overflowWrap: 'break-word',
-      whiteSpace: 'break-spaces',
+      overflowWrap: 'normal',
+      whiteSpace: 'normal',
       fontVariantLigatures: 'none',
       fontFeatureSettings: '"liga" 0',
       // the above doesn't seem to work in Edge
